assets/merging-fee.md

```diff
@@ -0,0 +1,63 @@
+# Merge Fee -- FAQ
+
+**Someone is asking for cypto payment for merging my Pull Request.  Is this a scam?**
+
+Always be cautious when someone asks you for payment.
+Check that there is a description about it under Trust Wallet Developers' doucmentation (this page).
+Check that the app is under the *trustwallet* github account, or that its owner belong to the trustwallet group.
+So no, this is not a scam.
+
+**Why?**
+
+Trust Wallet supports an insane number of tokens and coins.
+We are committed to support tokens used by our users.
+But supporting so many assets has a significant costs, in terms of running the infrastructure, ensuring continuous integrations,
+handling a huge number of requests to the assets info, etc.  A lot of spam/scam coins also find us.
+
+We have introduced this crypto contribution in order to filter down requests, so we have more time to handle genuine ones.
+
+**How  dare you ask for money in the case of an Open Source project?**
```

Comment:
Can we remove this section and use Curated Token Registry format: https://medium.com/@tokencuratedregistry/a-simple-overview-of-token-curated-registries-84e2b7b19a06 ?